### DISCUSSION TITLE
When using yarn workspaces (3.2.2): Query engine library for current platform could not be found.

### DISCUSSION THREAD

**Discussion by colleen-love** [prisma/prisma#14567] [Q&A] [answered]
I have the following setup
```
app
  \backend
  \web
```
The backend folder includes, well, the backend - including all of the prisma dependencies (version 4.1.1). Running yarn installs the prisma binaries in `app/node_modules/@prisma/engines`. 

That being said, prisma is looking in the following locations:
```
Searched Locations:

  C:\...\app\backend\.prisma\client
  C:\...\app\node_modules\@prisma\client
  C:\...\app\backend\.webpack\service
  C:\...\app\backend\.webpack\service\src
  C:\...\app\backend\.webpack\service\src
  /tmp/prisma-engines
  C:\...\app\backend\.webpack\service\src
```

How can I configure prisma to look in the engines folder?

**Comment by pantharshit00** [comment, marked as answer]
Hello @collinscangarella, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

If this is still relevant, I have put together a simple yarn workspaces with Prisma example here: https://github.com/harshit-test-org/prisma-yarn-workspaces-example

Let me know if this works for you.

**Reply by vladi-stevanovic** [reply]
👋 Hello @collinscangarella did you perhaps have a chance to check Harshit's recommendation? Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
